config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  root 'users#index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
```

Comment:
I suggest using `resources` because resource routing allows you to quickly declare all of the common routes for a given resourceful controller, and codes are cleaner. you can just do
```
resources :users, only: [:index, :show] do 
  resources :posts, only: [:index, :show] end
```